docs/man/docker-run.1.md

```diff
@@ -260,6 +261,25 @@ outside of a container on the host.
 **--sig-proxy**=*true*|*false*
    Proxy received signals to the process (non-TTY mode only). SIGCHLD, SIGSTOP, and SIGKILL are not proxied. The default is *true*.
 
+**--stdenv**=*true*|*false*
+   Provide information about the running container to that container via 
+environment variables.  By default (=*true*), the following environment
+variables will be set:
+
+   **CONTAINER_ID**: The long UUID identifier of the running container.
+   **CONTAINER_IMAGE**: The long UUID identifier of the image used for this container.
+   **CONTAINER_HOST**: The FQDN of the host on which this container is running.
```

Comment:
it might be useful to know the host IP addresses to which the container's ports are mapped.